src/components/OpeningHour.jsx

```diff
@@ -39,7 +39,9 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
     {label}
     {(nextTransition &&
      (!showNextOpenOnly || showNextOpenOnly && status === 'closed')) &&
-    ` - ${_('until {nextTransitionTime}', 'hour panel', { nextTransitionTime: nextTransition })}`
+    ` - ${_('reopening at {nextTransitionTime}', 'hour panel', {
+      nextTransitionTime: nextTransition,
+    })}`
```

Comment:
If status is `open`, the message would be wrong:
"Open - reopening at 20:00" , instead of "Open - until 20:00"

Reply:
Thank you @amatissart , I updated the pull request adding an element `NextTransition` to keep things more pleasant to read